hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/merkle/MerkleStakingInfo.java

```diff
@@ -83,6 +84,12 @@ public class MerkleStakingInfo extends PartialMerkleLeaf implements Keyed<Entity
     private long unclaimedStakeRewardStart;
     private long stake;
     private long[] rewardSumHistory;
+    // The consensus weight of this node in the network. This is computed based on the stake of this node
+    // at midnight UTC of the current day. If the stake of this node is less than minStake, then the
+    // weight is 0. Sum of all weights of nodes in the network should be less than 500.
+    // If the stake of this node A is greater than minStake,
+    // then A's weight is computed as (node A stake * 500/ total stake of all nodes).
```

Comment:
I would leave this part out, or change it to point to the weight calculation's implementation. Otherwise any changes in the implementation probably won't be reflected here

Reply:
yes changed. Thanks !